cli/internal/run/run.go

```diff
@@ -425,16 +429,14 @@ type RunOptions struct {
 	parallel bool
 	// Git diff used to calculate changed packages
 	since string
-	// Current working directory
-	cwd string
 	// Whether to emit a perf profile
 	profile string
 	// Force task execution
 	forceExecution bool
 	// Cache results
 	cache bool
 	// Cache folder
-	cacheFolder string
+	unresolvedCacheFolder string
```

Comment:
I think if we pass `cwd` into where we generate this struct, we can avoid having this unresolved.